acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/SimpleHttpClient.java

```diff
@@ -25,30 +25,43 @@
 import okhttp3.RequestBody;
 import okhttp3.Response;
 import okhttp3.ResponseBody;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class SimpleHttpClient {
-  private final OkHttpClient httpClient = new OkHttpClient();
+  private static final Logger LOG = LogManager.getLogger();
+  protected OkHttpClient httpClient;
   private static final okhttp3.MediaType JSON =
       okhttp3.MediaType.parse("application/json; charset=utf-8");
 
+  public SimpleHttpClient() {
+    httpClient = new OkHttpClient();
+  }
+
   public String get(final URI baseUrl, final String path) throws IOException {
     return this.get(baseUrl, path, Collections.emptyMap());
   }
 
   public String get(final URI baseUrl, final String path, Map<String, String> headers)
       throws IOException {
+    LOG.debug("GET {}, headers: {}", path, headers.toString());
     final URL url = baseUrl.resolve(path).toURL();
     final Request.Builder builder = new Request.Builder().url(baseUrl.resolve(path).toURL()).get();
     headers.forEach(builder::header);
-    final Response response = httpClient.newCall(builder.build()).execute();
-    assertThat(response.isSuccessful())
-        .describedAs(
-            "Received unsuccessful response from %s: %s %s",
-            url, response.code(), response.message())
-        .isTrue();
-    final ResponseBody body = response.body();
-    assertThat(body).isNotNull();
-    return body.string();
+    try {
+      final Response response = httpClient.newCall(builder.build()).execute();
+      assertThat(response.isSuccessful())
+          .describedAs(
+              "Received unsuccessful response from %s: %s %s",
+              url, response.code(), response.message())
+          .isTrue();
+      final ResponseBody body = response.body();
+      assertThat(body).isNotNull();
+      return body.string();
+    } catch (IOException e) {
+      LOG.error("GET {} failed", path, e);
+      return "";
+    }
```

Comment:
Hiding exceptions like this seems very weird. How do we tell the difference between a get that returns nothing and a get that fails?

Reply:
Empty data will still give a parse error, I can probably remove this section now that it's all handled, but when i was debugging a random exception at this point was very un-helpful...